Public/Src/Engine/Scheduler/Tracing/MultiExecutionLogTarget.cs

```diff
@@ -87,7 +90,15 @@ protected override void ReportUnhandledEvent<TEventData>(TEventData data)
         {
             foreach (var target in m_targets)
             {
-                data.Metadata.LogToTarget(data, target);
+                try
+                {

```

Comment:
Can we make this more targeted to just the cachemiss implementation so we don't potentially sweep other errors under the rug?